.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-pnpm run lint:fix && pnpm run prettier:fix && pnpm run test:api && npx jest --clear-cache && pnpm run e2e:api
\ No newline at end of file
+pnpm run lint:fix && pnpm run prettier:fix && pnpm run test:api && npx jest --clear-cache
```

Comment:
you can remove `pnpm run npx jest --clear-cache` aswell

Reply:
noted.